aspnet/mvc/models/validation.rst

```diff
@@ -1,8 +1,196 @@
+      </div>	  
+      </div>
+  </form>
+            
+Client-side validation prevents submission until the form is valid. The Submit button runs JavaScript that either submits the form or displays error messages. 
+
+IClientModelValidator
+---------------------
+You may create client side logic for your custom attribute, and JQuery's unobtrusive validation will execute it on the client for you automatically as part of validation. The first step is to control what values render in the ``data-`` attributes by implementing the ``IClientModelValidator`` interface as shown here:  
+
+.. literalinclude:: validation/sample/ClassicMovieAttribute.cs
+ :language: c#
+ :lines: 39-44
+ :dedent: 8
+ 
+Attributes that implement this interface are applied to model types or properties can add HTML attributes to generated fields. Examining the output for the ``ReleaseDate`` element reveals HTML that is similar to the previous example, except now there is a ``data-val-classicmovie`` attribute that was defined in the ``GetClientValidationRules`` method of ``IClientModelValidator``.
+
+.. code-block:: html
+   
+  <input class="form-control" type="datetime" 
+  data-val="true" 
+  data-val-classicmovie="Classic movies must have a release year earlier than 1960" 
+  data-val-required="The ReleaseDate field is required." 
+  id="ReleaseDate" name="ReleaseDate" value="" />
+
+JQuery's unobtrusive validation uses the data in the ``data-`` attributes to display error messages. However, jQuery doesn't know about rules or messages until you add them to jQuery's ``validator`` object. This is shown in the example below that adds a method named ``classicmovie`` containing custom client validation code to the jQuery ``validator`` object, then rules and errors. 
+
+.. code-block:: javascript
+   
+  $(function () {
+      jQuery.validator.addMethod('classicmovie',
+      function (value, element, params) {
+	      // custom validation code
+          return false;
+      }, '');
+
+      jQuery.validator.unobtrusive.adapters.add('classicmovie',
+      function (options) {
+          options.rules['classicmovie'] = {};
+          options.messages['classicmovie'] = options.message;
+      });
+  }(jQuery));
+
+Now jQuery has the information to execute the custom JavaScript validation as well as the error message to display if that validation code returns false. 
+
+Remote valiation
+----------------
+Remote validation is a great feature to use when you need to validate data on the client against data on the server. For example, your app may need to verify whether an email or user name is already in use, and it must query a large amount of data to do so. Downloading large sets of data for validating one or a few fields consumes too many resources. It may also expose sensitive information. An alternative is to make a round-trip request to validate a field; however, doing so diminishes the user's experience, especially when other fields quickly pass client side validation.
```

Comment:
> however, doing so diminishes the user's experience, especially when other fields quickly pass client side validation.

Suggest removing this last bit. It sounds too negative, like you shouldn't be using remote validation at all.